e2e-tests/specs/all.e2e.js

```diff
@@ -387,17 +380,21 @@ describe('Filter and Search Behaviors', async () => {
         await clearSearch.click();
         const cardRes = await $('.consonant-LoadMore-text').getText();
         expect(cardRes).toEqual('Showing 5 of 6 cards');
+        const newUrl = await browser.getUrl();
+        expect(newUrl).toEqual(`${serverPath}/html/e2e/e2e-grid.html?state=${state}&ch_Topic=Stock%252CPersonalization`);
     });
-    it('MWPW-137141: can clear a singluar selected filter from checkbox', async () => {
+    it('MWPW-137141: can clear a singluar selected filter from checkbox and see the updated results', async () => {
         const cloneConfig = structuredClone(config);
         const state = btoa(JSON.stringify(cloneConfig));
         const url = `${serverPath}/html/e2e/e2e-grid.html?state=${state}&sh_=Ze&ch_Topic=Stock%252CPersonalization`;
         await browser.url(url);
-        await browser.setTimeout({ script: 600000000 });
+        await browser.setTimeout({ script: 500000000 });
```

Comment:
@sheridansunier I guess this long pause is not intentional.

Reply:
lol no i'll fix that